hiroki/html-google-homepage/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="origin" name="referrer" />
+    <link rel="stylesheet" href="./stylesheet.css" />
+    <title>Google</title>
+  </head>
+  <body>
+    <main>
+      <header>
+        <div>
+          <ul>
+            <li>
+              <a href="https://mail.google.com/mail/u/0/#inbox"> Gmail </a>
```

Comment:
You add a tag with a correct way! 🎉 
Since you finish it 100%, let's go further.
Please search about `target="_blank"` and `rel="noopener noreferrer"` and try to summary when we should add `target="_blank"` and `rel="noopener noreferrer"`.

Reply:
```suggestion
              <a href="https://mail.google.com/mail/u/0/#inbox">Gmail</a>
```
Please check the different of above